enterprise/server/testutil/buildbuddy_enterprise/buildbuddy_enterprise.go

```diff
@@ -29,12 +30,36 @@ var (
 )
 
 const (
-	DefaultConfig = "enterprise/config/test/buildbuddy.selfauth.yaml"
-	NoAuthConfig  = "enterprise/config/test/buildbuddy.noauth.yaml"
+	DefaultConfigName = "test/buildbuddy.selfauth.yaml"
+	NoAuthConfigName  = "test/buildbuddy.noauth.yaml"
 )
 
+var (
+	// set by x_defs in BUILD file
+	// contains multiple paths separated by white-space
+	configFilesRunfilePaths string
+	buildbuddyRunfilePath   string
+
+	// set these variables from configFilesRunfilePaths during init()
+	DefaultConfigRunfilePath string
+	NoAuthConfigRunfilePath  string
+)
+
+func init() {
+	for _, p := range strings.Split(configFilesRunfilePaths, " ") {
+		if strings.HasSuffix(p, DefaultConfigName) {
+			DefaultConfigRunfilePath = p
+			continue
+		}
+		if strings.HasSuffix(p, NoAuthConfigName) {
+			NoAuthConfigRunfilePath = p
+			continue
+		}
+	}
```

Comment:
To simplify this a little, could we export the test config files as individual BUILD targets, rather than including them in the `config_files` glob? The `config_files` target gets distributed with our release binaries so it'd be nice to remove the testonly configs from that glob pattern anyway.

Reply:
I didn't like this pattern as well. I worked on https://github.com/bazelbuild/rules_go/pull/3852 as an alternative solutions instead. I think in the future we gonna have use cases for `rlocationpaths` on multiple files so it would be nice to have it ready.

I don't mind adding a `BUILD` file inside `enterprise/config/test`. Let me try that.